### ISSUE TITLE
develop Aviation Flight Rule graphics

### ISSUE THREAD

**Issue by cshartsough** [NOAA-GSL/adb_graphics#31]
based on this NCL method:

Aviation Flight Rules

visvar = grb_file->$VISNAME$(:,:) / 1000. ; convert to km
visvar = visvar * 0.62137 ; convert to miles
ceilvar = grb_file->$CEILNAME$(:,:) - grb_file->$GHSFCNAME$(:,:)
ceilvar = where (ismissing(ceilvar), -0.1, ceilvar)
ceilvar = (ceilvar * 3.28)
;
load "$NCL_HOME/obj/get_date_str.ncl"
;
; Loop to determine the Flight Rule Category
;
tvar = new((/ny,nx/),"float")
tvar = 0.0 ; set to VFR initially
tvar = where(ceilvar.ge.1000 .and. ceilvar.lt.3000,1.0,tvar) ; set tp MVFR
tvar = where(visvar.ge.3 .and. visvar.le.5,1.0,tvar)
tvar = where(ceilvar.ge.500 .and. ceilvar.lt.1000,2.0,tvar) ; set to IFR
tvar = where(visvar.ge.1 .and. visvar.lt.3,2.0,tvar)
tvar = where(ceilvar.lt.500 .and. ceilvar.gt.0,3.0,tvar) ; set to LIFR
tvar = where(visvar.lt.1,3.0,tvar)
;
tvar = where(tvar.eq.1.0,1.01,tvar)
tvar = where(tvar.eq.2.0,2.01,tvar)

So basically, read in the ceiling (which needs the surface height subtracted) and visibility, and convert them to miles and feet, respectively.  Then set up a new zero field, and assume VFR to be defined as zero.  Then at grid points where ceiling is 1000-3000ft, set values to 1.0 (MVFR).  MVFR can also be reached if visibility is between 3-5 miles.  IFR is achieved when ceiling is between 500-1000ft or visibility is between 1-3 miles.  LIFR happens when ceiling is between 0-500ft or visibility is less than 1 mile.  The last two statements were necessary because of a threshold issue (I think).

**Comment by cshartsough** [comment]
This task was completed in PR55 (September 2020)
